packages/@okta/vuepress-theme-prose/assets/css/components/_pricing.scss

```diff
@@ -194,6 +194,14 @@ $magenta: #e22866;
         display: none;
       }
 
+      &.desktop {
+        .additional-note {
```

Comment:
Is this actually fixable by tinkering with the vertical alignment/line height for the checkmark? I'm noticing that it actually seems to sit a bit high of center in the screenshots so maybe that's actually the issue.

Reply:
Yeah good catch, I've made the fix with vertical align now and updated the screenshot.